dockerfiles/enclave.Dockerfile

```diff
@@ -1,31 +1,39 @@
-FROM ghcr.io/edgelesssys/ego-dev:latest
+FROM ghcr.io/edgelesssys/ego-dev:latest AS build-base
```

Comment:
Why have the intermediate `build-base` step ? Can these steps just exist in the `build-enclave` ?

Reply:
There is no real need for it, just seems cleaner this way where stages only do smaller bits of work.

Reply:
Repeated, but having a top header explaining the stages would be awesome :)